lib/routes/callback/query.js

```diff
@@ -41,9 +41,9 @@ const registerRateLimitedCallback = ({
                 ELSE callback_request_total + 1
                END
       WHERE   id = ${registrationId}
-        AND   (    last_callback IS NULL 
+        AND   (    last_callback IS NULL
                 OR last_callback <= CURRENT_TIMESTAMP - ${`${rateLimitSeconds} secs`}::INTERVAL
-                OR (last_callback > CURRENT_TIMESTAMP - ${`${rateLimitSeconds} secs`}::INTERVAL AND callback_rate_count < ${rateLimitRequestCount})
```

Comment:
Not sure what effect this will have - do you mean to change this line?

Reply:
It should not have any effect, but leaving it in feels redundant. 

That is, if the `last_callback` is not null and not `<= CURRENT_TIMESTAMP` it surely is `> CURRENT_TIMESTAMP` so we can avoid a condition and only check `callback_rate_count`.

I guess it's mostly cosmetic.

Reply:
Don't think we need rate limt time period and rate limit count. Rate limit time period should be sufficent